themes/Paradox-QuietLight.tmLanguage.json

```diff
@@ -0,0 +1,391 @@
+// This is absolutely a work in progress. I can't properly see how everything is supposed to be tokenised until all effects/triggers/everything is defined properly.
+// Plus maybe some adjustments to the basic tokenisation are in order - but that's, again, only after everything is done at least for the first pass.
+// For now it's just a light theme that doesn't necessarily look like crap, just a bit of a colour mismatch everywhere. So that's a good thing, I hope.
+{
+	"name": "CKII - Quiet Light",
```

Comment:
Make the name Paradox, it'll work in all the games :)

Reply:
I just followed the Stellaris' theme convention :)